<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LOG_FORM</class>
 <widget class="QWidget" name="LOG_FORM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="Stack_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>491</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Login_Page">
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>0</y>
       <width>391</width>
       <height>501</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame {	
	background-color: rgb(255,192, 128);	
	color: rgb(220, 220, 220);
	border-radius: 10px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>221</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Viner Hand ITC</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(24, 14, 0);
	border-style: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="RES.qrc">:/RES/logo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>210</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255,255,255);
	color: rgb(24, 12, 0);
	border-style: none;
	border-radius: 10px;
	</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>310</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255,255,255);
	color: rgb(24, 12,0);
	border-style: none;
	border-radius: 10px;
	</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Viner Hand ITC</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(24, 14, 0);
	border-style: none;</string>
      </property>
      <property name="text">
       <string>Email Or Username:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Viner Hand ITC</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
	color: rgb(24, 14, 0);
	border-style: none;

	</string>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>370</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(24,14,0);
	color: rgb(255, 255, 255);
	border-style: none;
	border-radius: 10px;
	text-align: center;</string>
      </property>
      <property name="text">
       <string>L O G I N  I N</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>370</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(24,14,0);
	color: rgb(255, 255, 255);
	border-style: none;
	border-radius: 10px;
	text-align: center;</string>
      </property>
      <property name="text">
       <string>Click here for register</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_5">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>430</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(24,14,0);
	color: rgb(255, 255, 255);
	border-style: none;
	border-radius: 10px;
	text-align: center;</string>
      </property>
      <property name="text">
       <string>Forget Password</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Register_Page">
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>381</width>
       <height>501</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame {	
	background-color: rgb(255,192, 128);	
	color: rgb(220, 220, 220);
	border-radius: 10px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>210</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255,255,255);
	color: rgb(24, 12, 0);
	border-style: none;
	border-radius: 10px;
	</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Fullname</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>260</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255,255,255);
	color: rgb(24, 12, 0);
	border-style: none;
	border-radius: 10px;
	</string>
      </property>
      <property name="placeholderText">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>300</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255,255,255);
	color: rgb(24, 12, 0);
	border-style: none;
	border-radius: 10px;
	</string>
      </property>
      <property name="placeholderText">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>340</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255,255,255);
	color: rgb(24, 12, 0);
	border-style: none;
	border-radius: 10px;
	</string>
      </property>
      <property name="placeholderText">
       <string>Repeat Password</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>380</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(24,14,0);
	color: rgb(255, 255, 255);
	border-style: none;
	border-radius: 10px;
	text-align: center;</string>
      </property>
      <property name="text">
       <string>S I G N  U P</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>211</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Viner Hand ITC</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(24, 14, 0);
	border-style: none;</string>
      </property>
      <property name="text">
       <string>Coffee Shop</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>430</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(24,14,0);
	color: rgb(255, 255, 255);
	border-style: none;
	border-radius: 10px;
	text-align: center;</string>
      </property>
      <property name="text">
       <string>Already have an account? Click here</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="RES.qrc"/>
 </resources>
 <connections/>
</ui>
